freon@UshakovVasilii_Github.yahoo.com/extension.js

```diff
@@ -1,34 +1,31 @@
-const St = imports.gi.St;
-const PanelMenu = imports.ui.panelMenu;
-const PopupMenu = imports.ui.popupMenu;
-const Main = imports.ui.main;
-const Util = imports.misc.util;
-const Mainloop = imports.mainloop;
-const Clutter = imports.gi.Clutter;
-const Gio = imports.gi.Gio;
-const GLib = imports.gi.GLib;
-const GObject = imports.gi.GObject;
-
-const ExtensionUtils = imports.misc.extensionUtils;
-const Me = ExtensionUtils.getCurrentExtension();
-
-const SensorsUtil = Me.imports.sensorsUtil.SensorsUtil;
-const FreeipmiUtil = Me.imports.freeipmiUtil.FreeipmiUtil;
-const LiquidctlUtil = Me.imports.liquidctlUtil.LiquidctlUtil;
-
-const NvidiaUtil = Me.imports.nvidiaUtil.NvidiaUtil;
-const BumblebeeNvidiaUtil = Me.imports.bumblebeeNvidiaUtil.BumblebeeNvidiaUtil;
-const AticonfigUtil = Me.imports.aticonfigUtil.AtiConfigUtil;
-
-const Udisks2Util = Me.imports.udisks2.UDisks2;
-const HddtempUtil = Me.imports.hddtempUtil.HddtempUtil;
-const SmartctlUtil = Me.imports.smartctlUtil.SmartctlUtil;
-const NvmecliUtil = Me.imports.nvmecliUtil.NvmecliUtil;
-
-const FreonItem = Me.imports.freonItem.FreonItem;
-
-const Gettext = imports.gettext.domain(Me.metadata['gettext-domain']);
-const _ = Gettext.gettext;
+const Mainloop = imports.mainloop
```

Comment:
Maybe replace Mainloop calls with setTimeout and clearTimeout and drop this import?
Requires replacing the function calls at line 165, 431, 436, 461, 462, I can't make suggestions on those lines for some reason

Reply:
I found out that the GLib Interface also exposes timeout functions, so I changed to those instead.